lib/AppInfo/Application.php

```diff
@@ -46,7 +46,9 @@ public function register() {
 		Util::connectHook('OC_Filesystem', 'preSetup', $this, 'addStorageWrapper');
 
 		$request = $this->getContainer()->getServer()->getRequest();
-		if (!\OC::$CLI && strpos($request->getPathInfo(), '/login') !== 0
+		if (!\OC::$CLI
+			&& strpos($request->getPathInfo(), '/login') !== 0
+			&& strpos($request->getPathInfo(), '/apps/user_saml') !== 0
```

Comment:
What about the 404 pages? For example `/s/lajksdhflkjasdfkljasdfkljasdflkjad`?

Reply:
Right, let me copy the logic from the notification app. Makes more sense.